Meadowlark-js/packages/meadowlark-core/src/handler/DescriptorLoader.ts

```diff
@@ -126,7 +126,8 @@ async function loadParsedDescriptors(descriptorData: XmlDescriptorData): Promise
       const documentInfo: DocumentInfo = descriptorDocumentInfoFrom(descriptorDocument);
 
       const upsertRequest: UpsertRequest = {
-        id: documentIdForDocumentInfo(resourceInfo, documentInfo),
+        id: getDocumentUuidForDocument(),
+        meadowlarkIdentity: documentIdForDocumentInfo(resourceInfo, documentInfo),
```

Comment:
Let's call this `meadowlarkId` instead of `meadowlarkIdentity`. We use "Identity" to mean the un-hashed elements of a document, and "id" to mean the hashed version.